src/styles/DropDownMenu.ts

```diff
@@ -19,6 +19,29 @@ const sharedListStyles = {
   margin: `${Spacing.xSmall}px ${Spacing.Small}px`,
 };
 
+const sharedDateRangeStyles = {
+  display: 'inline-block',
+  position: 'absolute' as const,
+  right: `3rem`,
+};
+
+export const DateRangeRaw = {
+  Light: {
+    ...sharedDateRangeStyles,
+    color: Colors.SecondaryText,
+  },
+
+  Dark: {
+    ...sharedDateRangeStyles,
+    color: Colors.SecondaryTextDark,
+  },
+};
+
+export const DateRange = {
+  Light: style(DateRangeRaw.Light),
+  Dark: style(DateRangeRaw.Dark),
+};
```

Comment:
I have a lot of comments about the styles.

### 1. About the styles themselves.

I'm not a big fan of the absolute positioning. I tried to remove them locally, and this actually gives a good result: without the absolute positioning the ranges will show inside the dropdown button too! This is good!

We have a good tool to achieve the same behavior without using absolute positioning: use flex with `justify-content: space-between`!

Therefore my suggestion is to add a flex wrapper around the text and the display of the date ranges.

### 2. How and where to specify it?

I see you tried to follow the style we're using in a lot of places. Unfortunately this style isn't super great and is very verbose. Instead we can use MUI's own theming mechanism, that _already_ supports dark mode automatically. As you can see in https://github.com/mozilla/perfcompare/blob/main/src/theme/protocolTheme.ts, `text.secondary` will point to `SecondaryText` or `SecondaryTextDark` automatically depending on the theme.

How to use it? One way is with `sx` prop in all MUI components. Another way is with the `styled` function provided with MUI (we don't use that one yet but it's a valid option -- very similar to the use of `typestyle` in its result).

You need a MUI component to use it, this doesn't work with `<span>`. But you can use `<Box>` that inserts a `<div>` (the `component` props can change the actual HTML element but a `div` works fine if we use `flex` like suggested in point 1):

```js
<Box sx={{ color: "text.secondary",  ... other properties }}> ... </Box>`
```

It is much simpler because you don't need to do the `if` condition for the dark mode, etc, and all the code related to dark mode is in one place.

Of course as you've probably noticed, the project doesn't use this trick everywhere, and sometimes it would need some signifcant change to be able to do it because it's not as straightforward as using `text.secondary`, so the way you did isn't bad either, it's just more complicated than what it could be with our current state.